src/server/scripts/Spells/spell_dk.cpp

```diff
@@ -635,6 +637,70 @@ class spell_dk_glyph_of_scourge_strike_script : public SpellScript
     }
 };
 
+static constexpr uint32 SPELL_VISUAL_ID_HOWLING_BLAST = 66812;
+
+// 49184 - Howling Blast
+// 237680 - Howling Blast
+class spell_dk_howling_blast : public SpellScript
+{
+    PrepareSpellScript(spell_dk_howling_blast);
+
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellInfo({ SPELL_DK_HOWLING_BLAST_AREA_DAMAGE, SPELL_DK_FROST_FEVER });
+    }
+
+    void FilterTargets(std::list<WorldObject*>& targets)
+    {
+        targets.remove_if([&](WorldObject const* target)
```

Comment:
what does the sniff say about SPELL_GO hit targets for 49184

Reply:
only the expl target